chiselFrontend/src/main/scala/chisel3/core/Bits.scala

```diff
@@ -78,6 +78,15 @@ abstract class Element extends Data {
   */
 private[chisel3] sealed trait ToBoolable extends Element {
 
+  /** Casts this $coll to a [[Bool]]
```

Comment:
Is `$coll` meant to be substituted?

Reply:
@seldridge added this when he improved the ScalaDoc, I just copied it from `toBool`